Add recipe for garlic toast (#370)
LukeSmithxyz/based.cooking: src/garlic-toast.md

# Garlic Toast

Garlic toast, perfect as a substitute for crackers in a cheese platter.

- ⏲️ Prep time: 5 min

> - 🍳Cook time: 10-15 min

Reviewer: ```suggestion
- 🍳Cook time: 10-15 min (or 1 minute on broil)
```

## Ingredients

> - Sourdough loaf (sliced)

Reviewer: ```suggestion
- Any kind of firm, thin, sliced bread (Sourdough, Baguette)
```

> - Garlic-infused olive oil

Reviewer: ```suggestion
- Butter or Olive Oil (Optionally garlic-infused oil)
```
Butter also tastes good!

Author: you're probably not wrong, but it would hardly be garlic toast without the garlic-infused olive oil!

- Garlic salt

## Directions

> 1. Preheat oven to 180°C (356°F).

Reviewer: ```suggestion
1. Preheat oven to 180°C (350°F).
```
Typical mapping from C to F in cooking is 180C = 350C

> 2. Place sliced sourdough on a baking tray, prepared with aluminium foil

Reviewer: ```suggestion
2. Place sliced bread on a baking tray, covered with aluminum foil

```

Reviewer 2: `aluminium` is actually the correct spelling for most of the world. I think Canada and the US may be the only places that don't. So I guess the question is, do we want to force American spelling.

> 3. Generously coat sourdough slices with garlic-infused olive oil.

Reviewer: ```suggestion
3. Generously coat the bread slices with olive oil.
```

4. Season with garlic salt, to taste.

> 5. Bake for 10-15 minutes at 180°C (356°F) or until crunchy throughout.

Reviewer: ```suggestion
5. Bake for 10-15 minutes until crunchy throughout.
   - Alternatively set on broil for 1 minute.
```

6. Serve with cheese, charcuterie, pickles, dips, or to accommodate a soup. 

## Contribution

- Jack Crawford - [website](https://github.com/jjcrawford)

> 

Reviewer: ```suggestion
- Edited by Zyansheep

```
